runtime/syntax/nftables.yaml

```diff
@@ -1,28 +1,25 @@
 filetype: nftables
 
 detect:
-    filename: "nftables.conf$"
+    filename: "((nftables.conf$)|(nftables.rules.d/)|(nftables.d/)|(nftables.rules$))"
     header: "^(#!.*/(env +)?nft( |$)|flush +ruleset)"
-    
+
 rules:
     - type: "\\b(chain|counter|map|rule|ruleset|set|table)\\b"
     - type: "\\b(ether|inet|i(cm)?p(x|(v?(4|6))?)|tcp|udp)\\b"
-    - special: "\\b(elements|hook|policy|priority|type)\\b"
-    - identifier: "\\b(ct|iif|iifname|meta|oif|oifname|th)\\b"
-    - statement: "\\b(accept|drop|goto|jump|log|masquerade|reject)\\b"
+    - special: "\\b(elements|hook|policy|priority|type|state)\\b"
+    - identifier: "\\b(ct|iif|iifname|meta|oif|oifname|th|dport|sport|saddr|daddr)\\b"
+    - statement: "\\b(accept|drop|goto|jump|log|masquerade|reject|limit|queue)\\b"
     - preproc: "\\b(add|define|flush|include|delete)\\b"
-    - symbol: "([-=/:;,@]|[!=])"
-    - symbol.operator: "[<>.&|^!]|\\b(and|ge|gt|le|lt|or|xor)\\b"
-    - constant.string: '([\"]{1})(.*)([\"]{1})'
-    - constant.string: 
-         start: "(['\"])" 
-         end: "(['\"])" 
+    - symbol.operator: "[<>.&|^!=:;,@]|[!=]|\\b(and|ge|gt|le|lt|or|xor)\\b"
+    - constant.string: '([\+\-\*/][\"]{1})(.*)([\"]{1})'
+    - constant.string:
+         start: "\""
+         end: "\""
       # Integer Constants
-    - constant.number: "(\\b([1-9][0-9]*|0[0-7]*|0[Xx][0-9A-Fa-f]+|0[Bb][01]+)([Uu][Ll]?[Ll]?|[Ll][Ll]?[Uu]?)?\\b)" 
-      # Decimal Floating Constants 
-    - constant.number: "(\\b(([0-9]*[.][0-9]+|[0-9]+[.][0-9]*)([Ee][+-]?[0-9]+)?|[0-9]+[Ee][+-]?[0-9]+)[FfLl]?\\b)" 
+    - constant.number: "(\\b([1-9][0-9]*|0[0-7]*|0[Xx][0-9A-Fa-f]+|0[Bb][01]+)([Uu][Ll]?[Ll]?|[Ll][Ll]?[Uu]?)?\\b)"
       # Hexadecimal Floating Constants 
-    - constant.number: "(\\b0[Xx]([0-9A-Za-z]*[.][0-9A-Za-z]+|[0-9A-Za-z]+[.][0-9A-Za-z]*)[Pp][+-]?[0-9]+[FfLl]?\\b)" 
+    - constant.number: "(\\b0[Xx]([0-9A-Za-z]*[.][0-9A-Za-z]+|[0-9A-Za-z]+[.][0-9A-Za-z]*)[Pp][+-]?[0-9]+[FfLl]?\\b)"
```

Comment:
Why `Hexadecimal Floating Constants` and are you sure that float works here...as mentioned for the decimal values?
Additionally this comment has a trailing white space.

Reply:
> Additionally this comment has a trailing white space.
I removed the white space, that's because it is red

> Why Hexadecimal Floating Constants and are you sure that float works here...as mentioned for the decimal values?
Okay i've tried with it in a nftables file with the nft-checker (nft -cf nftables.conf) and nft wasn't happy with this. so i simplified it to normal decimal numbers, without any dot, or something else.